.github/workflows/add-lockdown-label.yml

```diff
@@ -11,6 +16,8 @@ permissions:
 jobs:
   add-label:
     runs-on: ubuntu-latest
+    # Only run on the main repository, not forks
+    if: github.repository == 'dotnet/sdk'
```

Comment:
This works. Alternate approach we used for the issue-labeler is to only check the org and also to allow a manual dispatch to bypass this check. I doubt this repo would be renamed or that you care to let forks manually dispatch the action though.

https://github.com/dotnet/runtime/blob/0ad494ba0eb84ada521d259c7d24dd0892c7a54d/.github/workflows/labeler-predict-pulls.yml#L49-L50